android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/FormUtils.java

```diff
@@ -2100,6 +2100,14 @@ public BufferedReader getRulesFromRepository(@NonNull Context context, @NonNull
         }
 
         ClientFormContract.Model clientForm = clientFormDao.getActiveClientFormByIdentifier(localeFormIdentity);
+        if (clientForm == null) {
+            // Strip anything before the '/'
+            if (StringUtils.isNotBlank(fileName) && fileName.contains("/") && !fileName.endsWith("/")) {
+                localeFormIdentity =  localeFormIdentity.split("/")[1];
+            }
+            //retry with just the filename without the file path prefix
+            clientForm = clientFormDao.getActiveClientFormByIdentifier(localeFormIdentity);
```

Comment:
I think we can put this inside the if block to avoid having 2 ClientFormDao calls if the form is not available and doesn't match the format `rule/rule-file-name`